packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -0,0 +1,223 @@
+// @flow
+
+import * as React from "react";
+import ReactDOM from "react-dom";
+import {StyleSheet} from "aphrodite";
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import {getClickableBehavior} from "@khanacademy/wonder-blocks-core";
+import DropdownCore from "./dropdown-core.js";
+import type {Item, DropdownItem} from "../util/types.js";
+import ActionItem from "./action-item.js";
+import OptionItem from "./option-item.js";
+import DropdownAnchor from "./dropdown-anchor.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * The child function, that takes in state the state of the element and
+     * returns an Element which will become the opener element for the dropdown.
+     */
+    children: (state: ClickableState) => React.Element<any>,
+
+    /**
+     * Can be used to override the state of the Dropdown by parent elemnents
+     */
+    opened?: boolean,
+
+    /**
+     * The items present in the Dropdown
+     */
+    menuItems: Array<Item> | Item,
+
+    /**
+     * A callback that returns items that are newly selected. Use only if this
+     * menu contains select items (and make sure selectedValues is defined).
+     */
+    onChange?: (selectedItems: Array<string>) => mixed,
+
+    /**
+     * The values of the items that are currently selected. Use only if this
+     * menu contains select items (and make sure onChange is defined).
+     */
+    selectedValues?: Array<string>,
+
+    /**
+     * Whether this menu should be left-aligned or right-aligned with the
+     * opener component. Defaults to left-aligned.
+     */
+    alignment?: "left" | "right",
+
+    /**
+     * Test ID used for e2e testing.
+     */
+    testId?: string,
+
+    /**
+     * Optional styling to add to the dropdown wrapper.
+     */
+    dropdownStyle?: StyleType,
+|};
+
+type ClickableState = {|
+    pressed: boolean,
+    hovered: boolean,
+    focused: boolean,
+|};
```

Comment:
It would be nice to create a shared type for this in a follow up PR.  Can you create a ticket in Jira for this and then add a `// TODO(WB-???): create a shared type for this` comment?